### DISCUSSION TITLE
Checkbox instead of title in DataGrid

### DISCUSSION THREAD

**Discussion by madcoda9000** [microsoft/fluentui-blazor#770] [Q&A] [answered]
Hello,

is it possible to insert a checkbox instead a title on a DataGrid Column?

![image](https://github.com/microsoft/fluentui-blazor/assets/25504739/badf0f78-4ff0-49a5-9204-8f3a7ef77f22)

Best Regards
Sascha

**Comment by vnbaaij** [comment, marked as answer]
Of course! 😉

You can use the `HeaderCellItemTemplate` parameter for that like this:

```
<PropertyColumn Title="..." Property="@(c => c.Selected)">
        <HeaderCellItemTemplate>
                <FluentCheckbox Value="false"  @onclick="SelectAll"/>
        </HeaderCellItemTemplate>
</PropertyColumn>
```

**Reply by madcoda9000** [reply]
Awesome! Thx for your help.

Best Regards 
Sascha
